app/client/components/noProduct.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { WithProductType } from "../../shared/productTypes";
+import { SupportTheGuardianButton } from "./supportTheGuardianButton";
+
+export interface NoProductProps extends WithProductType {
+  inTab: boolean;
+  supportRefererSuffix: string;
+}
+
+export const NoProduct = (props: NoProductProps) => (
+  <div>
+    <h2>You do not currently have a {props.productType.friendlyName}.</h2>
+    {props.inTab ? (
+      <p>
+        To manage your existing contribution or subscription, please select from
```

Comment:
Hmm actually, doesn't this text still imply that the user is on the membership tab?

Reply:
fair, but this is just a subtle prompt to navigate to other tabs, will review with Gus and amend in a future PR if required